library/Booster/SMT/Interface.hs

```diff
@@ -205,8 +217,10 @@ getModelFor ctxt ps subst
                                 Map.partition isLeft
                                     . Map.map (valueToTerm transState)
                                     $ Map.compose (Map.fromList pairs) freeVarsToSExprs
+                            untranslated =
+                                Map.mapWithKey (const . Var) untranslatableVars
```

Comment:
Unsure about this but I thought it might be helpful to indicate in the response that some potentially-relevant variables are not included.